ext/opcache/jit/zend_jit_trace.c

```diff
@@ -6063,6 +6063,18 @@ static const void *zend_jit_trace(zend_jit_trace_rec *trace_buffer, uint32_t par
 							goto jit_failure;
 						}
 						goto done;
+					case ZEND_FRAMELESS_ICALL_0:
+						jit_frameless_icall0(jit, opline);
+						goto done;
+					case ZEND_FRAMELESS_ICALL_1:
+						jit_frameless_icall1(jit, opline, op1_info);
+						goto done;
+					case ZEND_FRAMELESS_ICALL_2:
+						jit_frameless_icall2(jit, opline, op1_info, op2_info);
```

Comment:
Insert `op2_info = OP2_INFO();` before the call to avoid passing a potential uninitialized or stale value.